components/authentication/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LoginApiServiceImpl.java

```diff
@@ -127,22 +127,24 @@ public Response loginAppNamePost(String appName
                     String part1 = accessToken.substring(0, accessToken.length() / 2);
```

Comment:
Shall we use a meaningful variable name here?

Reply:
Changed in 24550cc